daemon/src/lib.rs

```diff
@@ -52,6 +52,8 @@ pub const HEARTBEAT_INTERVAL: std::time::Duration = Duration::from_secs(5);
 
 pub const N_PAYOUTS: usize = 200;
 
+pub const SETTLEMENT_INTERVAL: time::Duration = time::Duration::hours(24);
```

Comment:
Some docs would be nice on this!

Reply:
done! let me know if that's what you had in mind